modules/default-values/assets/js/editor/handlers/global-values.js

```diff
@@ -57,7 +65,9 @@ export default class GlobalValues extends BaseHandler {
 		// (default globals should be empty and not directly assign to the control setting)
 		element.settings.__globals__ = Object.fromEntries(
 			Object.entries( element.settings.__globals__ )
-				.filter( ( [ key ] ) => ! newDefaultSettingsKeys.includes( key )
+				.filter( ( [ key, value ] ) =>
+					! newDefaultSettingsKeys.includes( key ) ||
+					value !== newDefaultSettings.__globals__[ key ]
```

Comment:
Maybe extract to variables so it'll be more readable?

Reply:
Done